### PR TITLE
Update consent language on the calculator

### PR DESCRIPTION
**Pull request by bishiguro**
## Description

We are updating our consent language on the calculator to ensure users know how we handle their data. This is needed ahead of storing calculator submission data in the PEP database.

- [Asana](https://app.asana.com/0/1207889960030798/1208469066725526/f)
- [Email/Data Opt-In Consent Language](https://www.notion.so/rewiringamerica/Email-Data-Opt-in-Consent-Language-1d3e15dec9564a329c4f5f9cc12c748e?pvs=4)
- [Slack thread](https://rewiringameri-g3x1100.slack.com/archives/C04RAH1KDJT/p1726782456687519)

**Translation questions**:
- **I am not a Spanish speaker.**  I saw that Veekas used Deepl in a previous [PR](https://github.com/rewiringamerica/embed.rewiringamerica.org/pull/124) updating the terms language, so that's what I used here to translate the new copy. Do we have a more formal translation process?
- Because of the links I split the copy into multiple strings. The Spanish translation appears to correspond with the English such that think I can translate each of these substrings and combine them, but I know that wouldn't necessarily work for other languages. Do we have an existing solution for cases like these that I should use?

## Changes
- Updates the language below the email field, where we currently have the language for our terms.
- Removes the privacy language at the top of the calculator to avoid user confusion.

## Test Plan

- Temporarily hard-coded `this.lang` in state-calculator.tsx to 'es' to verify that the Spanish translation renders (Is there a better way to test this locally?)
- Verified that the "terms" and "privacy policy" links redirect to the correct pages in a separate tab

### Current state

Screenshot 2024-10-03 at 6 36 05 PM

### New state (English)

Screenshot 2024-10-03 at 5 47 25 PM

### New state (Spanish)

Screenshot 2024-10-15 at 1 09 10 PM

### CODE REVIEW

**Review by cjohnsbrown — COMMENTED** [review]
@bishiguro there's a translation process listed in [CONTRIBUTING.md](https://github.com/rewiringamerica/embed.rewiringamerica.org/blob/main/CONTRIBUTING.md#internationalization). Did you follow that?

File: src/state-calculator-form.tsx [hunk 1]
```diff
@@ -110,16 +110,22 @@ const renderEmailField = (
     />
     <div className="mt-1 mx-3 text-color-text-secondary text-xsm leading-normal">
       <slot name="email-helper">
-        {msg(
-          'Get updates on incentives, rebates, and more from Rewiring America.',
-        )}{' '}
-        {msg('View our')}{' '}
+        {msg('By sharing your email, you agree to our')}{' '}
```
Comment (oyamauchi) [inline src/state-calculator-form.tsx:113]: This one should have a description to tell translators what word is going to be next:

```
msg('By sharing your email, you agree to our', { desc: 'will be followed by "terms", as in terms of service' })
```
```

File: src/state-calculator-form.tsx [hunk 2]
```diff
@@ -110,16 +110,22 @@ const renderEmailField = (
...
         <a
           className="text-color-action-primary font-medium"
-          href="https://rewiringamerica.org/terms-of-use"
+          href="https://www.rewiringamerica.org/terms"
           target="_blank"
         >
           {msg('terms')}
+        </a>{' '}
+        {msg('and to receive updates from Rewiring America.')}{' '}
+        {msg('We’ll store and protect your data in accordance with our')}{' '}
```
Comment (oyamauchi) [inline src/state-calculator-form.tsx:122]: Same, add string description here

File: translations/es.xlf [hunk 3]
```diff
@@ -781,6 +768,22 @@
 <trans-unit id="s4cc06ab146dba194">
   <source>Electric transportation</source>
 <target>Transporte eléctrico</target></trans-unit>
+<trans-unit id="sbb254e3763405cb1">
+  <source>By sharing your email, you agree to our</source>
+  <target>Al compartir tu correo electrónico, aceptas nuestros</target>
```
Comment (oyamauchi) [inline translations/es.xlf:773]: This is one risk of machine translation: we've decided in this UI to use the Spanish formal "you", but this is using the informal "you" --- "tu" should be "su", and possibly "aceptas" needs to be conjugated differently?

(I don't actually know enough Spanish to do the translations from scratch myself, but I can recognize issues like this)

**Review by oyamauchi — CHANGES_REQUESTED** [review]
We've generally been OK with machine translation, but done our best to get a fluent Spanish speaker to look at it before merging. Failing that, someone like me who doesn't really "know" Spanish but has some basic knowledge of its grammar?

For canned language like this, I've gotten good use out of [linguee.es](https://linguee.es), which matches up different-language versions of the same text on public websites. This way we can at least see if our phrasing is in wide use.

File: translations/es.xlf (hunk 3 above)
Comment (bishiguro) [reply translations/es.xlf]: Oof thank you for catching this.

**Comment by bishiguro** [comment]
I updated the screenshot in the PR description for the Spanish translated page.

File: src/i18n/strings/es.ts [hunk 4]
```diff
@@ -159,7 +158,9 @@ export const templates = {
   sb9e8a9bfc8bcf398: `Rhode Island`,
   sba0e115b257c6647: `una bomba de calor de fuente de aire`,
   sbafed52303428041: `https://homes.rewiringamerica.org/es/federal-incentives/home-efficiency-rebates`,
+  sbb254e3763405cb1: `Al compartir tu correo electrónico, aceptas nuestros`,
```
Comment (veekas) [inline src/i18n/strings/es.ts]: `aceptas` –> `acepta`

File: translations/es.xlf (hunk 3 above)
Comment (veekas) [reply translations/es.xlf]: yeah `acepta` should work here

more generally re: formal translations; we recently discussed being more strict with translations (i.e. always do a formal review before merge) whenever possible

File: src/i18n/strings/es.ts (hunk 4 above)
Comment (bishiguro) [reply src/i18n/strings/es.ts]: Thanks, this is my original, now outdated change -- I've updated to `acepta`!

**Review by oyamauchi — APPROVED** [review]
Assuming the Spanish is now correct (which I'm not qualified to judge), I think this is OK.

**Comment by bishiguro** [comment]
@oyamauchi @veekas Quentin caught and Anissa confirmed one mistake with my latest translation. I updated the first sentence to: _Al compartir su correo electrónico, **usted** acepta nuestros términos y recibir actualizaciones de Rewiring America._

Anissa reviewed the full Spanish translation [here](https://rewiringameri-g3x1100.slack.com/archives/C02BGPM3Q3D/p1729020523506229?thread_ts=1729019339.997079&cid=C02BGPM3Q3D). I think we're good to go.

Screenshot 2024-10-15 at 1 09 10 PM

**Review by veekas — APPROVED** [review]

**Pull request merged**